Lib/fontTools/ttLib/tables/C_F_F_test.py

```diff
@@ -0,0 +1,384 @@
+"""cffLib_test.py -- unit test for Adobe CFF fonts."""
+
+from __future__ import print_function, division, absolute_import, unicode_literals
+from fontTools.misc.py23 import *
+from fontTools.misc.testTools import parseXML
+from fontTools.misc.textTools import deHexStr, hexStr
+from fontTools.misc.xmlWriter import XMLWriter
+from fontTools.ttLib import TTFont, newTable
+import re
+import unittest
+
+
+cffXML = """
```

Comment:
In all the other test modules, we usually keep the TTX files in a `testdata` folder rather than embedded as strings in the test module itself, especially when they are long.